InvenTree/InvenTree/tasks.py

```diff
@@ -286,3 +293,60 @@ def send_email(subject, body, recipients, from_email=None, html_message=None):
         fail_silently=False,
         html_message=html_message
     )
+
+
+def check_for_migrations():
+    """Checks if migrations are needed.
+
+    If the setting auto_update is enabled we will start updateing.
+    """
+    # Test if auto-updates are enabled
+    if not get_setting('INVENTREE_AUTO_UPDATE', 'auto_update'):
+        return
+
+    from plugin import registry
+
+    executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
+    plan = executor.migration_plan(executor.loader.graph.leaf_nodes())
+
+    # Check if there are any open migrations
+    if not plan:
+        logger.info('There are no open migrations')
+        return
+
+    logger.info('There are open migrations')
+
+    # Log open migrations
+    for migration in plan:
+        logger.info(migration[0])
+
+    # Set the application to maintenance mode - no access from now on.
+    logger.info('Going into maintenance')
+    set_maintenance_mode(True)
+    logger.info('Mainentance mode is on now')
+
+    # Check if we are worker - go kill all other workers then.
+    # Only the frontend workers run updates.
+
+    # TODO
+    if True:
+        logger.info('Current process is a worker - shutting down cluster')
+
+    # Ok now we are ready to go ahead!
+    # To be sure we are in maintenance this is wrapped
+    with maintenance_mode_on():
+        logger.info('Starting migrations')
+        call_command('migrate', interactive=False)
```

Comment:
We could also call the
- 1. `inv update` command
- 2. seperate django commands
To make the full run after an update

Reply:
just migrations should be enough for now